src/containers/FileExplorer/FileExplorer.tsx

```diff
@@ -192,7 +187,7 @@ class MapModelToAssetForm extends React.Component<Props, State> {
       },
       limit: 1000,
     };
-    this.setState({ fetching: true, searchResults: [], imageUrls: [] });
+    this.setState({ fetching: true, searchResults: [] });
```

Comment:
Here it might be a good idea to check prevState, and call URL.revokeObjectURL() on all urls created for images?

Reply:
good point will fix!